lib/inquisition/outputter.rb

```diff
@@ -44,6 +38,12 @@ def setup_default
 
       private
 
+      OUTPUTTER_TYPES = {
```

Comment:
This constant still not private, `Inquisition:: Outputter::Loader::OUTPUTTER_TYPES` will return the value